shell/app/modules/project/pages/issue/gantt/index.tsx

```diff
@@ -210,10 +216,21 @@ const IssuePlan = () => {
       chosenIssueType: undefined,
     });
   };
+
+  const handleScreenChange = (value: boolean) => {
+    updater.isFullScreen(value);
+  };
+
+  const buttonEle = document.getElementsByClassName('top-button-group');
```

Comment:
how about put these code under a useEffect hook.
when unmount make sure the style is restored.